### ISSUE TITLE
Add option of auto Cron Job checking of trunk or integrated with failure

### ISSUE THREAD

**Issue by reconwireless** [reconwireless/freepbx-trunk-monitor#6]
Can create a cron job to run trunk monitor script or just integrate the trunkalert.agi file with the trunk monitoring within FreePBX.... or both.

**Comment by lgaetz** [comment]
There is a script here:
https://github.com/lgaetz/freepbx-trunk-monitor/blob/master/drdrew/monitor_trunk.php

You may already know about it, but in case you don't, check the function wotsup (line 117) you will see some code that can be used to poll your SIP trunks and get the status.  I was thinking to turn the script into a module but will never get around to it.

**Comment by reconwireless** [comment]
Yep, I was just going to use the agi script and make it real simple then I got ambitious when I saw your module and the monitor_trunk file.  I'm mainly playing around with this as a learning experience... working with FreePBX Devs all day that can do this stuff in their sleep I could probably ask for their help and they would have this hammered out in a couple of hours... or less... but then I wouldn't learn anything.
